.circleci/config.yml

```diff
@@ -1333,7 +1333,7 @@ jobs:
 
   publish:
     docker:
-      - image: circleci/openjdk:11
+      - image: cimg/openjdk:17.0
```

Comment:
this is for ci-release

Reply:
the interesting part is that the "setup_sbt" already installs java 17, but from the logs sbt is still using the 11 anyway.. so I think we might be able to do some clean up but I will have a look in a later PR.